spec/postgresql/migrate/migrate_create_table_with_default_proc_spec.rb

```diff
@@ -78,17 +78,11 @@
     subject { client }
 
     it {
-      expect(Ridgepole::Logger.instance).to receive(:warn).with(<<-EOS)
-[WARNING] No difference of schema configuration for table `users` but table options differ.
-  from: {:id=>:uuid, :default=>"uuid_generate_v1()"}
-    to: {:id=>:uuid, :default=>"uuid_generate_v4()"}
-      EOS
-
       delta = subject.diff(expected_dsl)
-      expect(delta.differ?).to be_falsey
+      expect(delta.differ?).to be_truthy
```

Comment:
@winebarrel May I change this spec? This spec was introduced in https://github.com/winebarrel/ridgepole/pull/92, which resolves https://github.com/winebarrel/ridgepole/issues/91, but I think it is not related with https://github.com/winebarrel/ridgepole/issues/91.